app/_data/docs_nav_kuma_dev.yml

```diff
@@ -19,13 +19,95 @@ items:
             url: "/introduction/how-kuma-works/#vm-and-k8s-support"
           - text: Kuma vs XYZ
             url: "/introduction/how-kuma-works/#kuma-vs-xyz"
-      - text: Deployments
-        url: /introduction/deployments/
       - text: Enterprise
         url: /introduction/enterprise/
       - text: Changelog
         absolute_url: true
         url: https://github.com/kumahq/kuma/blob/master/CHANGELOG.md
+  - title: Kuma in Production
+    group: true
+    items:
+      - title: Deployment topologies
+        group: true
+        items:
+          - text: Overview
+            url: /production/deployment/
+          - text: Standalone deployment
+            url: /production/deployment/stand-alone/
+          - text: Multi-zone deployment
+            url: /production/deployment/multi-zone/
+          - text: Default ports
+            url: /production/deployment/networking/
+      - title: Control plane deployment
+        group: true
+        items:
+          # - text: Kong Mesh license
+            # url: /production/
+          - text: Deploy a multi-zone global control plane
```

Comment:
@fabianrbz Some of the items in the nav that are down a level don't have a title that displays for the page (I think this is one of them) even though the file has a title in the meta data. Do you know how I can fix that?

Reply:
Same reason as above, we don't support so many nesting levels :(